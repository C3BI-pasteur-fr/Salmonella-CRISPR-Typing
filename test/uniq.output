>sample_query1_Aba16
gatcgatcgatgctacgtagctagctagctagctagtcgatgcatcacatcatcg-Aba1
6|&tacgatcgtacgtactgatcgtagctagctgactgatcgatcgtcacgatcggcatc
atcgt
