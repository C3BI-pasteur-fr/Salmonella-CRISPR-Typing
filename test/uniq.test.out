>query1 <unknown description>
aa-spacer1&tcccggg
